Themes/default/languages/Help.english.php

```diff
@@ -356,9 +356,9 @@
 	This setting does not function when local cookies are enabled.';
 $helptxt['globalCookiesDomain'] = 'When using subdomain independent cookies (global cookies), you can specify which domain should be used for them. This should, of course, be set to your main domain - for example, if you are using <em>forum.example.com</em> and <em>www.example.com</em>, the domain is <em>example.com</em> in this case. You should not put the <em>http://</em> or <em>https://</em> part in front of it.';
 $helptxt['secureCookies'] = 'Enabling this setting will force the cookies created for users on your forum to be marked as secure. Only enable this setting if you are using HTTPS throughout your site, otherwise it will mean no-one will be able to log in properly!';
-$helptxt['securityDisable'] = 'This <em>disables</em> the additional password check for the administration page. This is not recommended!';
+$helptxt['securityDisable'] = 'When enabled, this <em>disables</em> the additional password check for the administration page. This is not recommended!';
 $helptxt['securityDisable_why'] = 'This is your current password. (the same one you use to login.)<br><br>The requirement to enter this helps ensure that you want to do whatever administration you are doing, and that it is <strong>you</strong> doing it.';
-$helptxt['securityDisable_moderate'] = 'This <em>disables</em> the additional password check for the moderation page. This is not recommended!';
+$helptxt['securityDisable_moderate'] = 'When enabled, his <em>disables</em> the additional password check for the moderation page. This is not recommended!';
```

Comment:
```suggestion
$helptxt['securityDisable_moderate'] = 'When enabled, this <em>disables</em> the additional password check for the moderation page. This is not recommended!';
```

Reply:
Addressed in https://github.com/SimpleMachines/SMF2.1/pull/5778/commits/b68a6dcdc21ea1244e9011f7d62c7b20b3b54c50